npm-shrinkwrap.json

```diff
@@ -1140,7 +1367,6 @@
 			"version": "7.6.3",
 			"resolved": "https://registry.npmjs.org/@babel/traverse/-/traverse-7.6.3.tgz",
 			"integrity": "sha512-unn7P4LGsijIxaAJo/wpoU11zN+2IaClkQAxcJWBNCMS6cmVh802IyLHNkAjQ0iYnRS3nnxk5O3fuXW28IMxTw==",
-			"dev": true,
```

Comment:
I'm not sure why these changes got made. All I've done is `npm install` after adding the new packages. Hopefully this is all ok.